components/push/storage/src/db.rs

```diff
@@ -143,6 +143,17 @@ impl Storage for PushDb {
         )?;
         Ok(affected_rows == 1)
     }
+
+    fn update_native_id(&self, uaid: &str, native_id: &str) -> Result<bool> {
+        let affected_rows = self.execute_named(
+            "UPDATE push_record set native_id = :native_id WHERE uaid = :uaid",
+            &[
+                (":native_id", &native_id.to_owned()),
```

Comment:
`.to_owned()` is unnecessary here, passing `&native_id`/`&uaid` should work fine.